fs/seq_file.c

```diff
@@ -397,16 +397,33 @@ EXPORT_SYMBOL(seq_escape_mem_ascii);
 
 void seq_vprintf(struct seq_file *m, const char *f, va_list args)
 {
+	void *temp;
```

Comment:
We shouldn't be modifying this code, it's used across all of Linux and carrying a diff here is *very* expensive.  These functions set the overflow flag in the `seq_file`, we should be checking that and writing only a partial amount in the write.  We're producing the same amount of output as the x86 arch, so I don't know why we'd need these changes.

Reply:
This code has been removed in a new version of the PR.